framework/core/src/Database/Migrator.php

```diff
@@ -31,12 +31,14 @@ public function __construct(
         protected ConnectionInterface $connection,
         protected Filesystem $files
     ) {
-        if (! ($connection instanceof MySqlConnection)) {
-            throw new InvalidArgumentException('Only MySQL connections are supported');
+        $doctrine = $connection->getDoctrineConnection()->getDatabasePlatform();
+
+        if (! Type::hasType('timestamp')) {
```

Comment:
What does this do?

Reply:
fixes timestamp columns on sqlite migrations.